dbaas/drivers/redis.py

```diff
@@ -302,10 +302,31 @@ def get_configuration(self):
             )
 
         with self.redis(instance) as client:
-            return client.config_get()
+            config = client.config_get()
+
+            if 'client-output-buffer-limit' in config:
+                config_COBL = config['client-output-buffer-limit']
```

Comment:
What u think if we do all at once ?
import re
config_COBL = config['client-output-buffer-limit']
params = re.match('client-output-buffer-limit normal (.*) slave (.*) pubsub (.*)', confg_COBL)
normal, slave, pubsub = params.groups()
config['client-output-buffer-limit-normal'] = normal
config['client-output-buffer-limit-slave'] = slave
config['client-output-buffer-limit-pubsub'] = pubsub

Reply:
It doesn't work.